pkg/get.go

```diff
@@ -84,3 +87,57 @@ func GetAppNameAndIP(appLabel, appNS string, clients environment.ClientSets) (st
 	return PodList.Items[0].Name, PodList.Items[0].Status.PodIP, PodList.Items[1].Name, nil
 
 }
+
+// GetCPUUsage will return the CPU usage by pod or node depending upon resource type.
+func GetCPUUsage(validation bool, resourceName, namespace, resourceType string) (int, error) {
+
+	var out, stderr bytes.Buffer
+	if validation == true {
+		cmd := exec.Command("bash", "-c", `kubectl top `+resourceType+` `+resourceName+` -n `+namespace+` --no-headers | awk '{print$2}'`)
+		cmd.Stdout = &out
+		cmd.Stderr = &stderr
+		err = cmd.Run()
+		if err != nil {
+			klog.Infof(fmt.Sprint(err) + ": " + stderr.String())
+			klog.Infof("Error: %v", err)
+			return 0, errors.Errorf("Fail to get the cpu usage before chaos, due to {%v}", err)
+		}
+		output := strings.Trim(out.String(), "")
```

Comment:
Can we use strings.TrimSpace instead? It will do nothing as it contain nil substring